### DISCUSSION TITLE
any way of running html files

### DISCUSSION THREAD

**Discussion by rwg100** [3kh0/ext-remover#406] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
>>see good hack
>>cant run
>>see i can use an html file
>>school blocks the app i can use to run html files (text on chrome web store with the icon that looks like <t>)

>>ask for help here

**Comment by md4r2** [comment, marked as answer]
[Use this ](https://www.tutorialspoint.com/online_html_editor.php)

**Reply by rwg100** [reply]
ty
